firehose-source/README.md

```diff
@@ -0,0 +1,43 @@
+Spring Cloud Stream Firehose Sample
```

Comment:
I think we should standardize on using asciidoc rather than markdown for those READMEs, as this would ease potential inclusion in our doc toolchain. Also, has better support for variables (which would come handy for _e.g._ replacing versions, etc)